.github/workflows/build-plugin-archive.yml

```diff
@@ -25,7 +25,7 @@ on:
         type: string
       COMPOSER_ARGS:
         description: Set of arguments passed to Composer.
-        default: '--no-dev --no-scripts --prefer-dist --optimize-autoloader'
+        default: '--no-dev --prefer-dist --optimize-autoloader'
         required: false
         type: string
       PHP_VERSION:
```

Comment:
The description needs to be updated, I guess.

Reply:
Done